Dockerfile

```diff
@@ -15,7 +15,9 @@ RUN apt-get update && \
 
 RUN pixi install --frozen -e ${PIXI_ENV}
 
-RUN pixi run -e ${PIXI_ENV} playwright install
+USER root
```

Comment:
Is this required to install the dependencies? In that case, would that means that there is stuff installed outside the pixi environment? If so, we copy stuff to the production and we would be missing some of the requirements.

Did you try to run the example from the container created from this?

Reply:
Good catch, I address this in commits below